coordinator/client/zookeeper/connection.go

```diff
@@ -233,7 +237,7 @@ func toClientEvent(zkEvent <-chan zklib.Event) <-chan client.Event {
 
 // ChildrenW returns the children of the node at the given path and a channel of
 // events that will yield the next event at that node.
-func (c *Connection) ChildrenW(path string) (children []string, event <-chan client.Event, err error) {
```

Comment:
Just an aside--
I like the idea of instead of passing in a done channel, to instead to offer a StopWatch method on the the return on the watcher.  See https://golang.org/pkg/time/#Timer as an example.